src/signals/incident-management/containers/ReporterContainer/components/FeedbackStatus.tsx

```diff
@@ -1,10 +1,13 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2021 Gemeente Amsterdam
```

Comment:
datumaanpassing?

Reply:
Bij een import auto correct? Lijkt me een beetje overbodig toch. Heb niks veranderd. Als we straks een ES Lint gefixed hebben en automatisch alle warnings fixen zou je alle files handmatig moeten langshaan om het jaartal te veranderen...